public/js/post_action.js

```diff
@@ -26,8 +26,38 @@ jQuery(function ($) {
     }
   }
 
+  function scroll_action_card_scroll() {
+    const $boxoutScroll = $('.post-content').find('> #action-card-scroll').not('.hidden');
```

Comment:
The take action boxout block can also be added on pages, which don't have `.post-content` or `.share-buttons`, so this function a lot of console errors. Do we have a condition somewhere to add the changes/event listeners in posts only ?

Reply:
Added a check to execute these functions only when there's `.post-content` 🙂